connect-inject/handler.go

```diff
@@ -181,10 +220,10 @@ func (h *Handler) Mutate(req *v1beta1.AdmissionRequest) *v1beta1.AdmissionRespon
 	// Decode the pod from the request
 	var pod corev1.Pod
 	if err := json.Unmarshal(req.Object.Raw, &pod); err != nil {
-		log.Printf("Could not unmarshal request to pod: %s", err)
+		h.Log.Error("Could not unmarshal request to pod", "Error", err)
```

Comment:
do we use `Error` or `err` as the key elsewhere? `err` seems more idiomatic to me

Reply:
The existing hclogging in the connect injector was already using `Error`, so I matched it. But you're right, it looks like the other components are using `err` so I'll switch it.